### ISSUE TITLE
Stop the repeating words!

### ISSUE THREAD

**Issue by VickiLanger** [VickiLanger/LGBTQ-of-the-day-bot#7]
**Is your feature request related to a problem? Please describe.**
Some nouns, adjectives, labels are being asked repeatedly

**Describe the solution you'd like**
I'd like the nouns, adjectives, labels to not be repeated so often

here's a similar solution with less moving parts than this repo https://github.com/VickiLanger/code-questions-bot/pull/24

**Describe alternatives you've considered**

read docs on random() to see if there is a better way
remove each nouns, adjectives, labels from the list until they've all been asked, then add them back in
track nouns, adjectives, labels being used in a database

**Additional context**
This repo used logs to stop repeating things. Maybe do it like they did
https://github.com/joaquinlpereyra/twitterImgBot/blob/master/twitterbot.py
https://github.com/joaquinlpereyra/twitterImgBot/blob/master/logs/logger.py

---
*Don't forget to follow the bot on [Twitter](https://www.twitter.com/LGBTQotd) to see the hard work you put in.*

**Comment by Apoorve73** [comment]
I would like to work on this.

**Comment by VickiLanger** [comment]
@Apoorve73 Sounds great! Have fun. I'm happy to help and answer questions if need be.

**Comment by Apoorve73** [comment]
Yes! Thanks a lot ! :)

**Comment by sanjibansg** [comment]
I would like to work on this issue. Do assign if available.. @VickiLanger

**Comment by VickiLanger** [comment]
@Apoorve73 Are you still wanting to work on this? If so, let me know. If not, I'll hand it over to @kahanikaar.

**Comment by Apoorve73** [comment]
Yes! But I am having some trouble continuing that! Really sorry for that! You may handover that to @kahanikaar ! :)

**Comment by VickiLanger** [comment]
Thanks for letting me know @Apoorve7.

@kahanikaar, in the morning, when I get back to my computer I will assign you
